collections/Selden_supra/MS_Selden_Supra_53.xml

```diff
@@ -38,43 +37,177 @@
+                     </msItem>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=235"><title>DIMEV</title> 235</ref></bibl>
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 83v–98r)</locus>
+                     <author key="person_9902409">Thomas Hoccleve</author>
+                     <title key="work_4807">Dialogus cum amico</title>
+                     <incipit>And endid my compleinte in this manere <lb/>Don knockid at my chaumbre dore sore</incipit>
+                     <explicit>Wole I translate and that my gifte I hope <lb/>Shal pourge as clene as keuerchiefs dooþ sope</explicit>
+                     <finalRubric>Explicit dialogus &amp; incipit quedam fabula quadam bona &amp; nobili Imperatrice Romana</finalRubric>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=508"><title>DIMEV</title> 508</ref></bibl>
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 98v–116r)</locus>
+                     <author key="person_9902409">Thomas Hoccleve</author>
+                     <title key="work_4810">Tale of the Emperor Gerelaus</title>
+                     <incipit>In romain actis writen is thus <lb/>Somtime an emperour in the citee</incipit>
+                     <explicit>And to in this wise and manere it seith <lb/>Wiche to that tale is good be knytte in feith</explicit>
+                     <finalRubric>here endith þe Prolog and begynneþ þe moralizing of þe <note>(remainder under the binding)</note></finalRubric>
+                     <note>Ends imperfectly due to loss of folios.</note>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=2621"><title>DIMEV</title> 2621</ref></bibl>
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 117r–133r)</locus>
+                     <author key="person_9902409">Thomas Hoccleve</author>
+                     <title key="work_4804">Ars sciendi mori</title>
+                     <incipit>Nowe herkene a doctrine substancial <lb/>ffirste howe lerne to die telle wole I</incipit>
+                     <explicit>That it may haue þe lyf þat hath noon ende</explicit>
+                     <finalRubric>Here endith to lerne to die⸝ And bigynneth a prolog on þe ixᵉ leson þat is redde on alle halwen day</finalRubric>
+                     <note>Missing stanzas 1–3.</note>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/record.php?recID=4879"><title>DIMEV</title> 4879</ref></bibl>
+                     <msItem>
+                        <locus>(fol. 133r)</locus>
+                        <incipit>To oþer þre partes⸝ edich in þe book</incipit>
+                        <explicit>That for to comprehende þat gladnesse <lb/>Verrely no wit may ne tounge expresse</explicit>
+                     </msItem>
```

Comment:
query what this is and whether the incipit is correct. DIMEV lists the incipit (https://www.dimev.net/Results.php?startsWith=T-t no. 425) but with a crossreference I can't make sense of. Does the print NIMEV help?

Reply:
DIMEV treat this prologue as part of no. 4879 – though I am not sure that is really correct.